parameters_template.yaml

```diff
@@ -11,14 +11,15 @@ network :
   eval :
     dense : 4
   policy :
-    dirichlet :
-    epsilon : 0.25
-    alpha : 0.03
+    dirichlet : 
+      epsilon : 0.25
+      alpha : 0.03
   loss :
     L2_norm : 0.001
   training :
     optimizer : momentum
     momentum : 0.9         # Momentum for SGD with mometum (MomentumOptimizer)
+	batch_size : 50
```

Comment:
I think this is a tab, which won't work in yaml.

Reply:
This yaml thing is going to be more work than its worth.